search/lib/convert/granules.js

```diff
@@ -92,6 +92,7 @@ function cmrGranToFeatureGeoJSON (event, cmrGran) {
     type: 'Feature',
     id: cmrGran.id,
     geometry: cmrSpatialToGeoJSONGeometry(cmrGran),
+    bbox: cmrGran.boxes,
```

Comment:
Does this have to be converted at all using `cmrBoxToGeoJsonPolygon()` above or is the format ok straight from the Granule?

Reply:
My thinking is that `cmrGran.boxes === bbox` and that `cmrBoxToGeoJsonPolygon()` would be converting it to a polygon from a bbox.

Reply:
Ok, cool. Then this looks good to me. Just needs Sr. approval and it's good to go.